assets/css/addPost.css

```
@@ -0,0 +1,109 @@
+.container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: none;
+    box-shadow: none;
+}
```

```suggestion
@import 'variables.css';

.container {
    display: flex;
    justify-content: center;
    align-items: center;
    border: none;
    box-shadow: none;
}
```
This is so you could use the variables defined in the variables.css